src/nnue/layers/affine_transform.h

```diff
@@ -63,6 +63,21 @@ namespace Eval::NNUE::Layers {
     // forward propagation
     const OutputType* Propagate(
         const TransformedFeatureType* transformed_features, char* buffer) const {
+
+  #if (defined(__MINGW32__) || defined(__MINGW64__)) && defined(USE_AVX2)
```

Comment:
This is presumably not a property of the MINGW platform but of the version of the compiler available on that platform... should the conditional be done differently ? Is there an open bug with gcc ?

Reply:
You are probably right.  I don't seem to have the issue with mingw/gcc 10.1.0 on windows.  I was simply trying to keep it the same as everywhere else in the code.  I'm not against changing it everywhere but probably better in a separate PR w/ good testing.

Reply:
I agree with doing this in a follow-up PR with good testing. I'll merge this based on the discussion in the PR mentioned above (adding a TODO for the define.).